render/topologies.go

```diff
@@ -224,9 +223,15 @@ var HostRenderer = MakeReduce(
 // PodRenderer is a Renderer which produces a renderable kubernetes
 // graph by merging the container graph and the pods topology.
 var PodRenderer = MakeReduce(
-	MakeMap(
-		MapContainer2Pod,
-		ContainerRenderer,
+	MakeSilentFilter(
+		func(n report.Node) bool {
+			_, isConnected := n.Latest.Lookup(IsConnected)
+			return n.Topology != Pseudo || isConnected
+		},
+		MakeMap(
+			MapContainer2Pod,
+			ColorConnected(ContainerRenderer),
+		),
```

Comment:
afaict, this will still fail with containers outside pods.

eg. if container1 connects to the internet, then both will be marked as connected, but container1 will be dropped by MapContainer2Pod.